### ISSUE TITLE
Patched weblogic being reported as vulnerable

### ISSUE THREAD

**Issue by arleybls** [johndekroon/serializekiller#8]
Have managed to patch few weblogics both 11g and 12c and the script still report them as vulnerable. Can you guys confirm if the detection logic is reliable?

**Comment by johndekroon** [comment]
You're correct, but there is no reliable way to confirm whether weblogic is patched or not (well, we could run the exploit, but hey, let's not do that :) ).
